### PR TITLE
Use env var for bundle analyzer webpack plugin

### PR DESCRIPTION
**Pull request by jparkrr**
I didn't know a better way of doing this besides the refactor

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -9,8 +9,72 @@ var APP_ENTRY = process.env.APP_ENTRY || "main";
 
 var THEME = process.env.THEME || "legacy";
 var THEME_DIR = path.resolve(__dirname, 'src/components/theme-' + THEME);
-const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+var BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```
Comment (schuyler1d) [inline webpack.config.js]: if we load it here, should it still be in devDependencies in package.json?

Comment (jparkrr) [reply webpack.config.js]: yep it should. And it's still in there from the react-loadable branch

Comment (schuyler1d) [reply webpack.config.js]: I meant more -- should it be moved to just 'dependencies' section since webpack won't build without it?

Comment (jparkrr) [reply webpack.config.js]: well, webpack itself plus a lot of things required for build is in devDependencies. The difference doesn't really matter anymore I think. Maybe we should move everything to regular dependences like CRA https://github.com/facebook/create-react-app/pull/2699#issuecomment-312281622

Comment (schuyler1d) [reply webpack.config.js]: ok, you've convinced me it doesn't matter, in any case. :+1:

**Pull request merged**